WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -327,7 +327,11 @@ public float getInterpolation(float t) {
         }
     }
 
-    private void refreshBlogDetails(@Nullable final Blog blog) {
+    private void refreshBlogDetails(@Nullable Blog blog) {
+        if (blog != null && blog.getAutomatedTransfer()) {
+            mBlogLocalId = WordPress.wpDB.getFirstVisibleAndNonAutomatedTransferBlogId();
+            WordPress.setCurrentBlog(mBlogLocalId);
```

Comment:
I still have to try this, but I think here if there is no other blog (i.e. the only existing blog is an automated transfer blog), then we're setting the current blog but then the UI is setup some lines below within this method, with the `blog` object populated. If the intention is to hide these sites, then we should add `blog = null;` right after `setCurrentBlog` in this `if` statement.